### DISCUSSION TITLE
Welcome to Lagoon Discussions!

### DISCUSSION THREAD

**Discussion by AlannaBurke** [amazeeio/lagoon#1822] [General]
In order to foster the open-source spirit of Lagoon, we've decided to join in the beta of GitHub discussions. We want current and potential users of Lagoon to be able to ask questions, talk about issues, and even post tips and tricks, without worrying that something might not be appropriate to post in the issue queue. In the past, we've used our [RocketChat](https://amazeeio.rocket.chat/home) for this kind of discussion, and while we'll continue to use that platform, having these discussions be public, persistent, and searchable, will be so valuable to this community!

Feel free to join in, ask a question, and get the discussions rolling! 

Please remember to abide by our [Code of Conduct](https://lagoon.readthedocs.io/en/latest/developing_lagoon/code_of_conduct/) when interacting here. Be patient and kind.

**Comment by ghost** [comment]


**Reply by tobybellwood** [reply]
(that link leads to internal team discussions - which are different)

Still waiting on some GH official docco about public discussions

**Reply by dan2k3k4** [reply]
Ah yes, you're right... my bad 😄 

And now I've just gone and deleted my comment as a test to see what happens and it's turned it into a `ghost` 🙈👻

**Reply by AlannaBurke** [reply]
Thanks for the thorough testing, team!

**Comment by anuraag4u** [comment]
While updating a post on **ghost website builder**  any random image I download from the internet is somehow interfering with the SSL certificate. I understand that it is due to **HTTP** and **HTTPS**  conflict. **Help Needed:** What should be the method so that it does not conflict and other useful insights if any. I hope I am in the right place. Thanks

**Reply by rocketeerbkw** [reply]
This is the discussion section for Lagoon, an open source docker build and deploy system for OpenShift & Kubernetes.

Unfortunately I don't think we can help you with your problems using [Ghost](https://ghost.org/), but I would suggest checking out their [help center](https://ghost.org/faq/) or [community forums](https://forum.ghost.org/).
